src/components/Graph.react.js

```diff
@@ -18,11 +19,14 @@ const filterEventData = (gd, eventData, event) => {
         const data = gd.data;
         for(let i=0; i < eventData.points.length; i++) {
             const fullPoint = eventData.points[i];
-            const pointData = filter(function(o) {
+            var pointData = filter(function(o) {
                 return !contains(type(o), ['Object', 'Array'])
             }, fullPoint);
-
-            if (has('customdata', data[pointData.curveNumber]) &&
+            if(data[0]['type'].toLowerCase() === 'pie'){
+              return omit(pointData, ['originalEvent', 'cxFinal', 'cyFinal', 'cy', 'cx', 'vTotal']);
```

Comment:
instead of "blacklisting" (excluding the things we don't want), can we "whitelist" (only include the keys that we want)? That way, it'll be a little bit more explicit